addon/modifiers/attach-qr-scanner.ts

```diff
@@ -9,49 +10,60 @@ import type ScannerService from 'ember-jsqr/services/ember-jsqr/-private/no-real
 import type JSQR from 'jsqr';
 import type { QRCode } from 'jsqr';
 
-type Args = {
-  positional: [MediaStream];
-  named: {
-    onData: <T>(data: string) => T;
-    onReady: <T>() => T;
-    highlightColor?: string;
-  };
-};
+interface Named {
+  [key: string]: unknown;
+  onData: <T>(data: string) => T;
+  onReady: <T>() => T;
+  highlightColor?: string;
+}
+
+type Positional = [MediaStream];
+
+export interface AttachQrScannerModifierSignature {
+  positional: Positional;
```

Comment:
```suggestion
  Positional: Positional;
```

Reply:
Yup, that's what I tried but it type error against that, like the Modifier types must be updated too, maybe. Going to try to update it too

Reply:
hm, I guess whatever works for now then -- this repo is quite a bit behind on things :sweat_smile: so I super appreciate you taking a poke